test/e2e/page-objects/pages/home/bitcoin-homepage.ts

```diff
@@ -0,0 +1,112 @@
+import HomePage from './homepage';
+
+class BitcoinHomepage extends HomePage {
+  protected readonly balance =
```

Comment:
:information_source:  in this case we cannot use the testId as suggested below, as we pass a text value in the below methods